content/en/examples/pods/storage/redis.yaml

```diff
@@ -4,11 +4,11 @@ metadata:
   name: redis
 spec:
   containers:
-  - name: redis
-    image: redis
+  - image: redis
```

Comment:
On other 3rd party blog sites on k8s and on k8s website itself, most of the time, name comes first before image key.